dietpi/dietpi-software

```diff
@@ -12759,6 +12805,47 @@ _EOF_
 
 		fi
 
+		software_id=191 # snapcast-client
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+			# Client IP needs to be added to allowed IP list
+			local snapcast_server_ip=
+			local snapcast_server_port=1704
+			local invalid_entry=
+			while :
+			do
+				if G_WHIP_INPUTBOX "${invalid_entry}Please enter the IP address of your snapcast server machine" && [[ $G_WHIP_RETURNED_VALUE =~ ^[0-9.]+$ ]]; then
+					snapcast_server_ip=$G_WHIP_RETURNED_VALUE
+					break
+				else
+					invalid_entry='[ERROR] Please enter a valid IP address. Please retry...\n\n'
+				fi
+			done
+
+			while :
+			do
+				G_WHIP_DEFAULT_ITEM=$snapcast_server_port
+				if G_WHIP_INPUTBOX "${invalid_text}PPlease enter the Port of your snapcast server machine\n - default: 1704" && disable_error=1 G_CHECK_VALIDINT "$G_WHIP_RETURNED_VALUE" 1
+				then
+					snapcast_server_port=$G_WHIP_RETURNED_VALUE
+					break
+				else
+					invalid_text='[ERROR] Please enter a valid port number. Please retry...\n\n'
+				fi
+			done
+
+			G_CONFIG_INJECT 'SNAPCLIENT_OPTS[[:blank:]=]' "SNAPCLIENT_OPTS=\"-h ${snapcast_server_ip} -p ${snapcast_server_port}\"" /etc/default/snapclient
+			# Restart snapclient so changes take effect.
+			G_EXEC systemctl restart snapclient
+
+		fi
+
+		software_id=192 # snapcast-server
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+			Banner_Configuration
+			G_WHIP_MSG 'Check the documentation on how to configure snapcast server audio sources\n\nThe config file is: /etc/snapserver.conf'
```

Comment:
Let's skip that here, to keep interactive input during installs at a minimum, so users can go, drink a coffee, come back after 20 minutes and all is finished and not haltet without an informational prompt 😃. We have the "Help!" option to show the docs URL and it is shown as well when confirming the install (is it? It should, I'll check 😄).

Reply:
Sure I'll remove it.

> I'll merge install and config blocks anyway with this or next release 😉.

I was wondering if the config was separate to allow a "reconfigure" option to go back through the steps of setting the IP / Port (in my case) without having to do a package reinstall. That would be quite powerful although I suspect hard to support.

Reply:
Fixed